submissions/KemiFalodun/HTML/login.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```

Comment:
Indentations are off on this file, please indent your tags properly. You can check the guide in the repo.